doc/js-api/remotestorage.rst

```diff
@@ -152,11 +200,29 @@ The following functions can be called on your ``remoteStorage`` instance:
 .. autofunction:: RemoteStorage#setCordovaRedirectUri(uri)
   :short-name:
 
+  Example::
+
+     remoteStorage.setCordovaRedirectUri('https://app.wow-much-app.com');
+
 .. autofunction:: RemoteStorage#startSync
   :short-name:
 
+  Example::
+
+     remoteStorage.startSync();
+
 .. autofunction:: RemoteStorage#stopSync
   :short-name:
 
+  Example::
+
+     remoteStorage.stopSync();
+
 .. autofunction:: RemoteStorage#onChange(path, handler)
   :short-name:
+
+  Example::
+
+     remoteStorage.onChange('/bookmarks/', function() {
```

Comment:
This is fired for directories? So it's different than the BaseClient change events?

Reply:
Let me check

Reply:
Yes, a directory as a path works, I tried it in myfavoritedrinks. That's what I understood from the current docs. It also works with a path to a document. Should I document both cases? I had only tried with a directory before

Reply:
Oh wait, I misread this. Directories are the normal use case here. But it needs to link to the extensive docs in BaseClient.

Reply:
Ack! Adding it

Reply:
You might also want to document that the handler function gets passed the change object in the example.